### ISSUE TITLE
Can you add infracost as hook?

### ISSUE THREAD

**Issue by lays147** [antonbabenko/pre-commit-terraform#206]
It would be nice to have this tool together with this hook suite.

https://www.infracost.io/

**Comment by antonbabenko** [comment]
I completely agree that it would be great to have it here but I don't think I have time to make it myself!

PR is welcome.
